### DISCUSSION TITLE
Undefined Sandbox Value Cap?

### DISCUSSION THREAD

**Discussion by Manouchehri** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#37] [General]
At the moment, the sandbox licenses don't have explicitly defined monetary on what is eligible.

https://github.com/MinistryFinancialServices/Virtual-Asset-Amendment-Consultation/blob/b78ab5b6a7b7ddbd9bde70cc377f633b0472ea6e/Virtual%20Asset%20(Service%20Providers)%20Act/03%20Part%203%20-%20Sandbox%20Licenses.txt#L104-L105

Just throwing ideas out there, maybe the cap could be a fixed amount, with a ± percentage based on the service provider's past volume of assets sold (and/or their current market value)?

**Comment by Aaron-XReg** [comment]
Good suggestion @Manouchehri. We are anxiously awaiting further details on the Sandbox regime! It is unclear whether the Sandbox provisions of the VASP Act will be commenced as part of these phase 2 amendments but we certainly hope they will. It would be a great opportunity to encourage innovative projects to be developed in the Cayman Islands.

**Reply by FinalTwist** [reply]
The cover note for the consultation states the Phase 2 will involve licensing for custodians and trading platforms - I assume sandbox and other provisions (like issuances) will be dealt with at a later time.

**Comment by AGIAI** [comment]
It likely makes sense for the sandbox to be based on a time period rather than just volume/asset.

This would allows for additional innovation

**Comment by CaraProvenance** [comment]
Would be great to see a further evolution of the Sandbox Licensing regime and practical implementation as well as clearer guidance on teh qualifying criteria for "innovative use of technology" in the VASP sector. 

Whilst there are certain limiting factors potentially with a sandbox license, it is a vehicle for greater learning, understanding and observation in a safe environment.

**Reply by Zhira7** [reply]
Should the qualifying criteria be really spelled out in legislation though ? Would that not  make it too narrow based on current use cases? The current wider wording seems to be appropriate to allow more future flexibility
